src/raft/uv_writer.c

```diff
@@ -207,7 +207,7 @@ static void uvWriterPollCb(uv_poll_t *poller, int status, int events)
 				req->status = RAFT_IOERR;
 				goto finish;
 			}
-			return;
+			continue;
 		}
 
 		uvWriterReqSetStatus(req, (int)event->res);
```

Comment:
Why is it that we don't call `uvWriterReqSetStatus` in `uvWriterAfterWorkCb`? Maybe it is not needed but I am not sure.

Reply:
It's done at the bottom of `uvWriterWorkCb`.